css/properties/alignment-baseline.json

```
@@ -12,7 +12,7 @@
           ],
           "support": {
             "chrome": {
-              "version_added": "≤80"
+              "version_added": "≤15"
```

See: https://github.com/caugner/css-properties/blame/chrome/properties.css

```suggestion
              "version_added": "1"
```